docs/src/pages/components/data-grid/group-pivot/group-pivot.md

```diff
@@ -6,6 +6,178 @@ title: Data Grid - Group & Pivot
 
 <p class="description">Use grouping, pivoting, and more to analyze the data in depth.</p>
 
+## Row grouping [<span class="plan-premium"></span>](https://mui.com/store/items/material-ui-pro/)
```

Comment:
```suggestion
## Row grouping [<span class="plan-premium"></span>](https://mui.com/store/items/material-ui-pro/)
```
Since this is a sort of superset of the tree view, should it come after that section? Or perhaps have its own page? (The page is quite long now).

Reply:
I don't see the row grouping as a superset of the Tree Data but as a different way of grouping
On the Tree Data, each rows gives it's position in the Tree, the groups are real rows.
On the Row Grouping, the position is deduced from the other rows, the groups are fake rows.
[AG Grid](https://www.ag-grid.com/javascript-data-grid/grouping/) is putting "Row Grouping" before, I think because it is the most complete / generic feature and the Tree Data doc refers a lot to the "Row Grouping" doc for shared behavior.

For the split, I agree that the page is long. I propose we wait for https://github.com/mui-org/material-ui/issues/30091 and then we split the big pages like the Filtering / Columns / Group Pivot pages.
Unless it's really to long for you in which case we can split it now. Just we won't have the additional depth on the sidebar to create a "Group & Pivot" section containing all the new pages extracted from the `group-pivot` page.

Reply:
Perhaps superset was the wrong word, but the point was that row grouping builds on the features of the tree view, so it makes sense to introduce that first to build foundational knowledge. (Not that people necessarily read the content in a linear manner, but at least to guide the reader if they happen to.

Reply:
Happy to wait for the restructure to decide what and where to split.

Reply:
Do we agree that what you call "tree view" is the Tree Data in the Data Grid doc ?

Reply:
Yes, sorry, mixing my terminology